sql/database.sql

```diff
@@ -8294,3 +8294,13 @@ CREATE TABLE `immunization_observation` (
   PRIMARY KEY (`imo_id`)
 ) ENGINE=InnoDB;
 -- --------------------------------------------------------
+
+
+-----------------------------------------------------------
+--add edit patient issue for_title new validation record build
+--
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES
+('lists', 'PageValidation', 'PageValidation', 298, 1, 0, '', '', '', 0, 0, 1, ''),
+('PageValidation', 'add_edit_issue', 'add_edit_issue.php', 0, 0, 0, '', '{form_title:{presence: true}}', '', 0, 0, 1, '');
```

Comment:
if this will not be included then the default validation of title hardcoded in the page will be deleted.